packages/block-library/src/site-logo/editor.scss

```diff
@@ -0,0 +1,37 @@
+.wp-block[data-align="center"] > .wp-block-site-logo {
+	margin-left: auto;
+	margin-right: auto;
+	text-align: center;
+}
+
+.wp-block-site-logo {
+	&.is-resized {
+		display: table;
+	}
+
+	.custom-logo-link {
+		cursor: inherit;
+
+		&:focus {
+			box-shadow: none;
+		}
+
+		&.is-transient img {
+			opacity: 0.3;
+		}
+	}
+
+	img {
+		display: block;
+		max-width: 100%;
+	}
+
+	.editor-media-placeholder__button {
+		margin-bottom: 0;
+	}
+
+	.block-editor-media-placeholder__cancel-button {
+		flex-basis: 100%;
+		text-align: center;
+	}
```

Comment:
I know it's not specific to this block, but altering styles of reusable components by targetting their classNames is not great. It's hard to find the right balance though

Reply:
You're right, and I don't think we need this anymore anyways so I've taken it out.